eox_tagging/api/v1/viewset.py

```diff
@@ -0,0 +1,54 @@
+"""
+Viewset for Tags.
+"""
+from django_filters import rest_framework as filters
+from rest_framework import viewsets
+from rest_framework.authentication import SessionAuthentication
+from rest_framework.permissions import IsAdminUser
+from rest_framework_oauth.authentication import OAuth2Authentication
+
+from eox_tagging.api.v1.filters import TagFilter
+from eox_tagging.api.v1.pagination import TagApiPagination
+from eox_tagging.api.v1.serializers import TagSerializer
+from eox_tagging.edxapp_wrappers import get_site
+from eox_tagging.models import Tag
+
+
+class TagViewSet(viewsets.ModelViewSet):
+    """Viewset for listing and creating Tags."""
+
+    serializer_class = TagSerializer
+    authentication_classes = (OAuth2Authentication, SessionAuthentication)
+    permission_classes = (IsAdminUser,)
+    pagination_class = TagApiPagination
+    filter_backends = (filters.DjangoFilterBackend,)
+    filter_class = TagFilter
+    lookup_field = "key"
+    http_method_names = ["get", "post", "delete", "head"]
+
+    def get_queryset(self):
+        """Restricts the returned tags."""
+        owner_type = self.request.query_params.get('owner_type', None)
+        include_invalid = self.request.query_params.get('include_invalid', None)
+        user = self.request.user
+        site = get_site()
+
+        if include_invalid in ["true", "1"]:
```

Comment:
I assume that included_invalid is a string, which could be "true" or "1", what happen with "True"

Reply:
right, I'll change that